openapi/nexus-internal.json

```diff
@@ -3062,7 +3062,6 @@
         ]
       },
       "DnsConfigParams": {
-        "description": "DnsConfigParams\n\n<details><summary>JSON schema</summary>\n\n```json { \"type\": \"object\", \"required\": [ \"generation\", \"time_created\", \"zones\" ], \"properties\": { \"generation\": { \"type\": \"integer\", \"format\": \"uint64\", \"minimum\": 0.0 }, \"time_created\": { \"type\": \"string\", \"format\": \"date-time\" }, \"zones\": { \"type\": \"array\", \"items\": { \"$ref\": \"#/components/schemas/DnsConfigZone\" } } } } ``` </details>",
```

Comment:
I assume these descriptions were coming from progenitor - do we care that they aren't being replaced by something else?

Reply:
Yeah these descriptions are what happens when you use a progenitor-generated type in a Dropshot server, effectively doubling up on progenitor. I don't think it's a huge problem.